.travis.yml

```diff
@@ -114,10 +116,16 @@ before_install:
   ## If we are building in debug, there are some tests to ignore.
   - if [ "$BTYPE" = "Debug" ]; then export TESTS_TO_EXCLUDE="$TESTS_TO_EXCLUDE|testInitState|testPrescribedForce|testInducedAccelerations|testAssemblySolver|testCMC|testOptimizationExample|testComponents|testForward"; fi
 
+  # Run superbuild to download, configure, build and install dependencies.
+  - mkdir $OPENSIM_DEPENDENCIES_BUILD_DIR
+  - cd $OPENSIM_DEPENDENCIES_BUILD_DIR
+  - cmake $TRAVIS_BUILD_DIR/dependencies -DCMAKE_INSTALL_PREFIX=$OPENSIM_DEPENDENCIES_INSTALL_DIR -DCMAKE_BUILD_TYPE=$BTYPE
+  - make -j$NPROC
```

Comment:
You don't need to use NPROC here; you can use 8. I use NPROC just when building wrapping since building the wrapping uses up a lot of memory.